enclave_build/src/lib.rs

```diff
@@ -103,8 +102,14 @@ pub struct Docker2Eif<'a> {
 }
 
 impl<'a> Docker2Eif<'a> {
+    /// If the 'docker_dir' argument is Some (i.e. --docker-dir flag is used), the docker daemon will
+    /// always be used to build the image locally and store it in the docker cache.
+    ///
+    /// If the 'oci_image' argument is Some (i.e. the --image-uri flag is used) and the '--docker-dir'
```

Comment:
Not sure this part of comment is relevant here: there is no `oci_image` argument here. probably you wanted to write about `docker_image` argument and pulling the image with a help of docker daemon?

Reply:
Left from older code, missed it.